### PR TITLE
Site redesign

### PR DESCRIPTION
**Pull request by hatzka-nezumi**
I hope my dedication to the commit naming gag is satisfactory. You could even call it... commitment?

### CODE REVIEW

**Review by wheals — COMMENTED** [review]
New images look mostly good, but using Slick for Felt and Feferi for Squiddles seem a little odd, and the Volume 4 image just looks weird to me. These are basically nitpicks though

**Review by wheals — COMMENTED** [review]
Generally not sure I'm happy about the move away from using the album covers themselves, but the ones that bother me the most I can just fix myself

**Comment by hatzka-nezumi** [comment]
I used Slick for the Felt because I couldn't find a good Felt-related image that had the red background (complementary colors and all). It's from `[S][I] ==>` (the Snowman flash), FWIW. Similarly, the Feferi shot is from `[S] Jade: Wake up.`, which I chose because it's the only use of anything from Squiddles! in the comic proper (aside from the sample in the flash version of Descend).

**Review by wheals — CHANGES_REQUESTED** [review]
Homestuck&CANWC MIDIs V11.rar has LEEKS and should be removed (we really should make a version that's generally usable outside the team)

File: index.html [hunk 1]
```diff
@@ -10,6 +10,12 @@
     a.c2 {font-weight: bold}
     span.c1 {color: red;}
     </style>
+    <style type="text/css">
+    a.c1 {font-weight: bold}
+    </style>
+    <style type="text/css">
+    a.c1 {font-weight: bold}
```
Comment (wheals) [inline index.html:17]: What's going on here?

Comment (wheals) [reply index.html:17]: Or actually, generally, why is stuff here instead of `style.css`

**Comment by hatzka-nezumi** [comment]
Oh, I didn't realize. I didn't look through it too closely, just made sure it was actually full of HS MIDIs and added it.

File: index.html (hunk 1 above)
Comment (hatzka-nezumi) [reply index.html:17]: I have _no idea_

**Review by wheals — CHANGES_REQUESTED** [review]
they should be removed before a merge then

File: index.html (hunk 1 above)
Comment (hatzka-nezumi) [reply index.html:17]: On further inspection I think that's Tidy trying to remove inline formatting. Good job little bot :) I'll move that over to `style.css`, one sec

**Review by wheals — APPROVED** [review]

**Pull request merged**